frontend/src/pages/learningCenter/DocTypeFilters.tsx

```diff
@@ -52,26 +53,18 @@ const DocTypeFilters: React.FC<DocTypeFiltersProps> = ({ categoryApps }) => {
 
   return (
     <FilterSidePanelCategory key="enabled-filter" title="Resource type">
-      {Object.keys(OdhDocumentType).map((docType) => {
-        const value = OdhDocumentType[docType as keyof typeof OdhDocumentType];
-        return (
-          <FilterSidePanelCategoryItem
-            data-id={value}
-            id={value}
-            key={value}
-            checked={docTypeFilters.includes(value)}
-            onClick={(e) =>
-              onFilterChange(
-                value,
-                (e.target as React.AllHTMLAttributes<HTMLInputElement>).checked || false,
-              )
-            }
-            title={docType}
-          >
-            {`${docType} (${docCounts[value]})`}
-          </FilterSidePanelCategoryItem>
-        );
-      })}
+      {Object.entries(OdhDocumentType).map(([docType, value]) => (
```

Comment:
the subsequent code references docType in `[docType, value]` though; how should I handle that? Since I think `enumIterator` only returns the value part of it